src/Components/Home.jsx

```diff
@@ -0,0 +1,55 @@
+import React, { useState } from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { Link } from 'react-router-dom';
+import { addCoin } from '../Redux/Coins/CoinSlice';
+
+const Home = () => {
+  const [query, setQuery] = useState('');
+  const dispatch = useDispatch();
+
+  const isLoading = useSelector((state) => state.coins.loading);
+  const allCoins = useSelector((state) => state.coins.data.coins);
+
+  return (
+    <div data-testid="home-1" className="">
+      <div className="w-[100%] flex justify-center items-center">
+        <form className="text-white flex items-center justify-center w-[90%] md:w-[60%] m-2">
+          <input onKeyUp={(e) => { setQuery(e.target.value.toLowerCase()); }} className="bg-[#0d0b13] px-4 py-2 outline-none w-[80%] border border-white/50 rounded-l-full focus:border-violet-600/50 focus:shadow-sm shadow-violet-400" type="text" placeholder="Search Coin name" required />
+          <input onClick={(e) => { e.preventDefault(); setQuery(query); }} className="py-2 px-4 bg-[#1a1729] border border-white/50 hover:bg-violet-400/75 sm:w-[20%] cursor-pointer rounded-r-full" type="button" value="Search" />
+        </form>
+      </div>
+      <div>
+        {isLoading ? <span className="text-center text-2xl flex items-center justify-center h-[40vh]">Loading ! Please Wait...</span> : ''}
+        <ul className="grid grid-cols-1 md:grid-cols-2 lg:grid-cols-3 xl:grid-cols-4">
+          {allCoins?.filter((coin) => (coin.name.toLowerCase().includes(query)))
+            .map((coinf) => (
+              <Link to="/coin" onClick={() => { dispatch(addCoin(coinf)); }} key={coinf.id}>
+                <li className="flex items-center gap-2 py-2 px-4 bg-violet-900/20 m-2">
+                  <img className="h-10" src={coinf.icon} alt={coinf.id} />
+                  <h2 className="md:text-xl">{coinf.name}</h2>
+                  <span className={coinf.priceChange1d < 0 ? 'text-red-400 ml-auto' : 'text-green-400 ml-auto'}>
+                    {coinf.priceChange1d}
+                    {' '}
+                    <i className={coinf.priceChange1d < 0 ? 'fa-solid fa-arrow-trend-down' : 'fa-solid fa-arrow-trend-up'} />
+                  </span>
+                </li>
+              </Link>
+            ))}
+        </ul>
+        {' '}
+        {!allCoins && isLoading === false ? (
+          <div className="text-center text-2xl flex items-center justify-center h-[40vh] ">
+            <span>
+              <span className="changer">Oh Snap !</span>
+              {' '}
+              We have a problem...
+            </span>
+          </div>
+        ) : ''}
+      </div>
```

Comment:
Nice and beuatiful design so far, however, as stated in the project requirements, you are suppose to follow the design guidelines including: 

- [ ] Colors (select one main color for your website).
- [ ] Typography: font face, size and weight.
- [ ] Layout: composition and space between elements.

There are some differences between yours and that of the actual design which you will need to fix.

1.  The home page is actually suppose to be in two rows with a light and dark color alternating between the cards on each column.  

2.  Kindly check the size and position of the country images, the country name and numbers below, as well as the small arrows on the top right corner of each country card. 


### Your design 

![Screenshot from 2023-02-07 18-48-55](https://user-images.githubusercontent.com/65233860/217340331-2dd6edd8-23d4-4541-81a3-e00409d12b18.png)


### The actual design

![Screenshot from 2023-02-07 18-49-55](https://user-images.githubusercontent.com/65233860/217340385-059426a9-e523-4e27-a9e5-5541970805a5.png)

Reply:
Thank you for your review @jerryowusu I have done my best to implement the requested style changes.
![Screenshot 2023-02-08 021052](https://user-images.githubusercontent.com/114587346/217360791-14ab761b-a123-45b8-b1e5-50526f5119c2.png)